pkg/cmd/roachtest/testdata/github/extra_parameters

```diff
@@ -0,0 +1,62 @@
+# Test that extra parameters added by the test are included in the github issue.
+
+add-failure name=(other)
+----
+ok
+
+add-param label=(MVT_deploymentMode) value=(separate-process)
+----
+ok
+
+post
+----
+----
+roachtest.github_test [failed]() on test_branch @ [test_SHA](). A Side-Eye cluster snapshot was captured on timeout: [https://app.side-eye.io/snapshots/1](https://app.side-eye.io/snapshots/1).
+
+
+```
+other
+```
+
+Parameters:
+ - <code>MVT_deploymentMode=separate-process</code>
```

Comment:
It breaks the symmetry... Since we have this historical prefix, `ROACHTEST_`, and since mixed version tests are technically embedded in the roachtest framework, `ROACHTEST_MVT_deploymentMode` seems reasonable.